modules/odc-creating-projects-using-developer-perspective.adoc

```diff
@@ -9,7 +9,7 @@ You can use the *Developer* perspective in the {product-title} web console to cr
 
 [NOTE]
 ====
-Projects starting with `openshift-` and `kube` host cluster components that run as Pods and other infrastructure components. As such, {product-title} does not allow you to create Projects starting with `openshift-` or `kube-` using the CLI. Cluster administrators can create these Projects using the xref:../../cli_reference/openshift_cli/administrator-cli-commands.adoc#new-project[`oc adm new-project` command].
+Projects starting with `openshift-` and `kube-`are considered critical by {product-title}. As such, {product-title} does not allow you to create projects starting with `openshift-` or `kube-` using the *Developer* perspective. Cluster administrators can create these projects using the `oc adm new-project` command.
```

Comment:
Hi @bobfuru we need to add a space between `kube-` and are. The formatting has gone awry cause of this problem. I have raised another PR to fix this in 4.3, 4.4, and 4.5. If you could add that space here for 4.2 it would be great. Also, request you to please merge the PR for this change for the other versions: https://github.com/openshift/openshift-docs/pull/23197
I am sorry I did not see this earlier :(

Reply:
No worries, @Preeticp! I have fixed the space issue and also made the update on line 8 (namespace -> cluster). Will merge this now, along with related https://github.com/openshift/openshift-docs/pull/23197.

Reply:
Thank you!!